### ISSUE TITLE
The maps app does not load on mobile chromium based browsers at all

### ISSUE THREAD

**Issue by szaimen** [nextcloud/maps#99]
The loading is finished, but the map is not loaded. There also isn't any any error-notification. See the screenshot below:
![20190830_171521](https://user-images.githubusercontent.com/42591237/64031976-d3531780-cb49-11e9-9fb1-096f7256ccd9.png)

I am using Brave 1.3.1, based on Chromium 76.0.3809.132 on Android 9. I checked it also with Chrome 76.0.3809.132 -> same result. Then I also tried Firefox Preview, aka "Fenix" and there it just loads fine.

**Comment by anaqreon** [comment]
Did you forget to run "make" from the maps app directory? You will need composer and npm for the make script to install the libraries that are not included in the git repo itself.

**Comment by szaimen** [comment]
@anaqreon I just installed it from the appstore. 
On every other platform (especially on Desktop-Chromium) everything works as espected.

**Comment by anaqreon** [comment]
I did not read your original post carefully enough to see that out loaded on Firefox. Sorry about that. We'll have to wait for others to test in that browser to see if it is reproducible.

**Comment by szaimen** [comment]
okay 👍

**Comment by eneiluj** [comment]
I can reproduce the issue with default LineageOS browser... No idea why though. Anybody know how to see browser logs with this one?

**Comment by szaimen** [comment]
I found this:
https://support.symantec.com/us/en/article.HOWTO81826.html
and this:
https://www.youtube.com/watch?v=9H6B-oIVYIA

**Comment by anaqreon** [comment]
I reproduced this bug with an Amazon Appstore installed Brave browser on Android. 

If your phone is rooted or you use ADB you can view logs. I can't do that at the moment though.

**Comment by eneiluj** [comment]
I think you both talk about system logs. Here we want the internal browser logs. Not sure it's possible with this browser.

**Comment by eneiluj** [comment]
The youtube video seems interesting.

**Comment by szaimen** [comment]
I think I found the debug logs:
```
script.js:6 Uncaught TypeError: window.navigator.registerProtocolHandler is not a function
    at HTMLDocument.<anonymous> (script.js:6)
    at u (main.js:39)
    at Object.fireWith [as resolveWith] (main.js:39)
    at Function.ready (main.js:39)
    at HTMLDocument.O (main.js:39)
```

**Comment by eneiluj** [comment]
I'm on it too. Same error with official Chrome Android browser. I'm fixing it right now :smile:.

**Comment by szaimen** [comment]
Very cool 👍😁

**Comment by eneiluj** [comment]
Ok it now loads correctly in Chrome base mobile browsers. The problem came from calling a function that did not exist: the one to be able to open "geo://" links with the browser in Maps. Not very useful since people most likely have a mapping app on their mobile which is able to open such links.

Thanks a lot!

It will be included in v0.1.2 which is coming soon.

**Comment by szaimen** [comment]
Nice to hear! 
Thank you, too! 
Now I can use the maps app on brave mobile 🎉🎉🎉
